ApollosProject/apollos-prototype
# Fixes sharing text feature

## DESCRIPTION

### What does this PR do, or why is it needed?

Updates the `share` function so it accepts a `message` property and fixes a typo that was preventing `message` from being passed down. 

### What design trade-offs/decisions were made?

Chose to now engage in a more full featured refactor like @redreceipt did in #935. Trying to find a way to get the intended functionality working without a full scale refactor. 

### How do I test this PR?

1. Open a content item with a text feature
2. Click the share button
3. click "copy to clipboard" 
3. paste somewhere else on your ios device (see screenshot for details) 

### What automated tests did you write?

## TODO:

- [x] Closes #934

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
